packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -254,6 +257,28 @@ class ExpansionTile extends StatefulWidget {
   ///   [ExpansionTileThemeData].
   final Color? collapsedTextColor;
 
+  /// Tile's border shape when the sublist is expanded.
+  ///
+  /// If this property is null then [ExpansionTileThemeData.shape] is used. If that
+  /// is also null then a [Border] with vertical direction sides default to [ThemeData.dividerColor] is used
+  ///
+  /// See also:
+  ///
+  /// * [ExpansionTileTheme.of], which returns the nearest [ExpansionTileTheme]'s
+  ///   [ExpansionTileThemeData].
+  final ShapeBorder? shape;
+
+  /// Tile's border shape when the sublist is collapsed.
+  ///
+  /// If this property is null then [ExpansionTileThemeData.collapsedShape] is used. If that
+  /// is also null then a [Border] with vertical direction sides default to Color [Colors.transparent] is used
```

Comment:
Same here, it also looks like there is a redundant Color

Reply:
in the expansion panel that we are using, there is no color (i.e, ThemeData.borderColor will lerp from opacity 00 to FF) when the panel is collapsed but with the new changes, we can assign color even when the panel is collapsed so we need to have a fallback color when there is nothing.

I've used `Color.transparent` so that the design will not break with the previous implementations.